src/App.tsx

```diff
@@ -15,71 +15,89 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [name, setName] = useState('Nothing');
```

Comment:
store here only name

The text you should render if nothing is selected: `No goods selected`